tests/linearGaussian_snle_test.py

```diff
@@ -502,3 +502,108 @@ def test_api_nle_sampling_methods(
         posterior.train(max_num_iters=10)
 
     posterior.sample(sample_shape=(num_samples,), show_progress_bars=False)
+
+
+@pytest.mark.slow
+@pytest.mark.parametrize("num_dim", (1, 2))
+@pytest.mark.parametrize("prior_str", ("uniform", "gaussian"))
+@pytest.mark.parametrize("model_str", ("zuko_maf", "zuko_nsf"))
+def test_c2st_nle_unconstrained_space(
+    num_dim: int, prior_str: str, model_str: str, mcmc_params_accurate: dict
+):
+    """Test SNL on linear Gaussian in unconstrained space.
+
+    Args:
+        num_dim: parameter dimension of the gaussian model
+        prior_str: one of "gaussian" or "uniform"
+
+    """
+    num_samples = 500
+    num_simulations = 3000
+    trials_to_test = [1]
```

Comment:
I think we can reduce the length of this test quite a bit by removing the dependency on the trials and just test with 1 trial. 
then we can construct `x_o` at the top here, and get the target samples in the same if-else as the prior and remove the for loop below.